src/main/java/io/citrine/jcc/search/core/result/BaseMultiSearchResult.java

```diff
@@ -55,17 +55,30 @@ public Long getTook() {
     }
 
     /**
+     * Set the list of successes.  This overwrites any successes that are already saved.
+     * @param successes List of successes.
+     * @return This object.
+     */
+    @JsonSetter("successes")
```

Comment:
I would prefer what I suggested only because it is consistent with the way that the rest of the library is set up.